### PR TITLE
Add Chinese translation for index docs

### PR DESCRIPTION
**Pull request by waynerv**
We (a small team) are working to complete the doc translation of Chinese version in the next two months.

### CODE REVIEW

**Comment by RunningIkkyu** [comment]
@waynerv I'm working on the Chinese trans too, may I have your wechat or email?

File: docs/zh/docs/index.md
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: 1. "Standard Python types"  is a proper noun, should be tranlated into `Python标准类型` not `标准的Python类型` .
2.  The relationship between `Pydantic` and `Standard Python types` should be parallel, not progressive。That could lead to ambiguity.

The translation should be 
```txt
我们借助 `Pydantic` 以及 `Python` 标准类型来声明请求体。
```

File: docs/zh/docs/index.md
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: Redundant space after "编辑器支持，".

File: docs/zh/docs/index.md
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: Missing a space after "基准测试".

**Review by RunningIkkyu — CHANGES_REQUESTED** [review]
Great translations! @waynerv

**Comment by waynerv** [comment]
@RunningIkkyu Great! You can send your wechat to my gmail: ampedee@gmail.com. I will contact you later.

File: docs/zh/docs/index.md
Comment (Dustyposa) [inline docs/zh/docs/index.md]: I can’t make sure this sentence which  be translated is clear.
“总之，你只需要声明参数，请求体等类型一次。和函数参数声明类型相同。” this one is better? need other review

File: docs/zh/docs/index.md
Comment (Dustyposa) [inline docs/zh/docs/index.md]: `HTTP Basic auth` is also called "HTTP 基本认证" in [wiki](https://zh.wikipedia.org/wiki/HTTP%E5%9F%BA%E6%9C%AC%E8%AE%A4%E8%AF%81).
It's can be translated.

File: docs/zh/docs/index.md
Comment (Dustyposa) [inline docs/zh/docs/index.md]: MIss space before “时安装”

**Comment by jamesyangget** [comment]
天哪，有翻译的人了，我也自己开了个[翻译项目](https://github.com/apachecn/fastapi-docs-cn)，我感觉自己在做无用功

**Comment by Dustyposa** [comment]
> 天哪，有翻译的人了，我也自己开了个[翻译项目](https://github.com/apachecn/fastapi-docs-cn)，我感觉自己在做无用功

has interested in join our team? my email zmzposa@outlook.com

**Comment by tiangolo** [comment]
This is amazing! Thanks everyone for doing this! :rocket: :earth_asia: :speech_balloon: 

And special thanks for the reviews from others, that's key here, as I don't know Chinese :sweat_smile: 

When others feel like this is complete, that there are no more corrections necessary, and hopefully that the style of "conversation" is similar, please add a new review approving the changes of this PR.

That way, after seeing that there's some consensus on the translation I'll be able to merge it. :tada:

**Review by RunningIkkyu — APPROVED** [review]

**Comment by tiangolo** [comment]
@Dustyposa do you think this is ready? You can let me know with a review approving it :smile: :+1: 

@phy25 if you have the chance, could you help me with a review here?

I'm just waiting for more reviews approving it before merging. :globe_with_meridians: :rocket:

**Comment by waynerv** [comment]
@tiangolo I can ask the Chinese community for some help. More and more Chinese developers have started to use FastAPI recently.

**Review by Dustyposa — APPROVED** [review]

**Review by lpdswing — APPROVED** [review]
checked, i approve this

**Review by RunningIkkyu — APPROVED** [review]

**Comment by tiangolo** [comment]
This is awesome! :rocket: :earth_asia: :bow: 

Thanks for all the work @waynerv ! :cake: :clap: :tada: 

Thanks for the reviews and comments @RunningIkkyu , @Dustyposa , @lpdswing , @jamesyangget :muscle: :nerd_face: :bow: :cake:

**Pull request merged**
